packages/shared/src/components/parts/graph.component.tsx

```diff
@@ -78,6 +78,16 @@ const CustomizedXAxisTick = ({
   );
 };
 
+const weekdayColors: Record<string, string> = {
```

Comment:
```suggestion
const weekdayColors: Record<typeof WEEK_DAYS[number], string> = {
```
このように書けそうです。importの追加が必要なので手元で反映してもらえると助かります :bow: